### PR TITLE
support count_true and count_false for boolean columns in BigQuery

### PR DESCRIPTION
**Pull request by suelai**
Allow `count_true` and `count_false` metrics for boolean columns for BigQuery
From my testing (I have blurred the table name):
Screenshot 2023-11-10 at 15 56 52

closes https://github.com/elementary-data/elementary/issues/1273

### CODE REVIEW

File: macros/edr/data_monitoring/monitors/monitors.sql [hunk 1]
```diff
@@ -4,6 +4,7 @@
       'column_any_type': ['null_count', 'null_percent', 'not_null_percent'],
       'column_string': ['min_length', 'max_length', 'average_length', 'missing_count', 'missing_percent', 'not_missing_percent'],
       'column_numeric': ['min', 'max', 'zero_count', 'zero_percent', 'not_zero_percent', 'average', 'standard_deviation', 'variance', 'sum']
```
Comment (noaKurman) [inline macros/edr/data_monitoring/monitors/monitors.sql:6]: You're missing a semicolon here

**Review by noaKurman — CHANGES_REQUESTED** [review]

File: macros/edr/data_monitoring/monitors/monitors.sql (hunk 1 above)
Comment (noaKurman) [reply macros/edr/data_monitoring/monitors/monitors.sql:6]: This is why the tests fail

Comment (suelai) [reply macros/edr/data_monitoring/monitors/monitors.sql]: @noaKurman Ah when I merged main into my branch, I missed it when resolving the conflicts. It is ready now!

**Comment by ofek1weiss** [comment]
Hey @suelai, sorry for the long delay 😓 in here now so lets do it 😎 

there seems to be a small issue [here](https://github.com/elementary-data/dbt-data-reliability/pull/610/files#diff-1210b27ed5be9a6d5ed751791c43cd58c5f8ff90a0a21d091c15cadd5aa0a7c5R67), you are missing a comma at the end of the line, like so:
```
{%- if 'count_false' in column_monitors -%} {{ elementary.count_false(column) }} {%- else -%} null {% endif %} as count_false,
```
after that i believe that everything should work and this could be merged 🙏

**Comment by suelai** [comment]
> Hey @suelai, sorry for the long delay 😓 Im here now so lets do it 😎
> 
> there seems to be a small issue [here](https://github.com/elementary-data/dbt-data-reliability/pull/610/files#diff-1210b27ed5be9a6d5ed751791c43cd58c5f8ff90a0a21d091c15cadd5aa0a7c5R67), you are missing a comma at the end of the line, like so:
> 
> ```
> {%- if 'count_false' in column_monitors -%} {{ elementary.count_false(column) }} {%- else -%} null {% endif %} as count_false,
> ```
> 
> after that i believe that everything should work and this could be merged 🙏

Great, fixed in https://github.com/elementary-data/dbt-data-reliability/pull/610/commits/c0a60a65ce8c3f039cd1a0845c04f22d48df895d

**Review by ofek1weiss — APPROVED** [review]
👍

**Review by noaKurman — APPROVED** [review]

**Comment by ofek1weiss** [comment]
@suelai thank you so much for this contribution :pray:! Keep your eyes peeled for the next release 👀

**Pull request merged**